.npmignore

```diff
@@ -1,2 +1,13 @@
 test/
-tmp/
\ No newline at end of file
+tmp/
+scripts/
+.documentup.json
+.gitignore
+.jshintrc
+.lgtm
+.travis.yml
+appveyor.yml
+RELEASE.md
+
+# This is commented out because I don't see the point of not including it.
+# MAINTAINERS
```

Comment:
I think we can keep the MAINTAINERS as well. We can probably delete these comments